docs/ssh-keys/create-ssh-key-pair.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Create an SSH key pair
+intro: Sometimes you may need to connect to your servers on a new device. Create SSH keys to authenticate your server access on the new device.
+links:
+    overview:
+    quickstart:
+    previous:
+        - /docs/ssh-keys
+    next:
+        - /docs/ssh-keys/add-ssh-key
+    guides:
+    related:
+        - /docs/ssh-keys/add-ssh-key
+        - /docs/ssh-keys/remove-ssh-key
+    featured:
+---
+
+1. In a terminal window, run command below, substituting in your Devopness email address
+    ```bash
+    ssh-keygen -t rsa -b 4096 -C "<your devopness email address>"
+    ```
+    > This creates a new SSH key, using the provided email as a label.
+    > Devopness is working to support the most populars cryptosystem keys types,
+    > for now we are are supporting RSA and DSA.
+1. Follow the prompts to complete the creation of the new SSH Key pair
+    > There should be 2 new files inside `$USER/.ssh` directory, `id_rsa` (private key) and `id_rsa.pub` (public key)
+1. Start the `ssh-agent`, using command:
+    ```bash
+    eval "$(ssh-agent -s)"
```

Comment:
1. `eval` is `evil`, please do not instruct users to use such a risk command, neither in bash nor in any other programming language.
A bit of context if needed: https://stackoverflow.com/questions/17529220/why-should-eval-be-avoided-in-bash-and-what-should-i-use-instead
2. I don't see a benefit of adding the complexity of using `ssh-agent` + `ssh-add` to to a simple connection to a remote server. i would recommend keep things straight to the point. We should instruct users towards the simplest possible path to connect to the servers. Advanced use cases, such as `ssh-agente + ssh-add` to avoid [re]typing the passphrase all the time is beyond the scope of Devopness documentation.

Reply:
I meant no security harm, it was the way I personally learn to use SSH reading [GitHub documentation](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent#adding-your-ssh-key-to-the-ssh-agent).

Until now, I thought that was the recommended path.

We can remove this step completely

Reply:
@jfoliveira text updated, removed `ssh-agent` and `ssh-add` mentions